core/src/main/scala/org/apache/spark/scheduler/TaskSetManager.scala

```diff
@@ -98,6 +98,14 @@ private[spark] class TaskSetManager(
   var totalResultSize = 0L
   var calculatedTasks = 0
 
+  override def maxShare = {
```

Comment:
Maybe specifying return types? (See Return types in https://cwiki.apache.org/confluence/display/SPARK/Spark+Code+Style+Guide)